source/docs/03-metadata.xml

```diff
@@ -1382,156 +1382,315 @@
-                     <head>Cutouts</head>
+               <div xml:id="sourcesHandwritten" type="div4">
+                     <head>Additional Elements for Describing Handwritten Sources</head>
+                     <p></p>
                      <p>
                         <specList>
-                           <specDesc key="cutout"/>
+                           <specDesc key="handList"/>
+                           <specDesc key="inscription"/>
+                           <specDesc key="explicit"/>
                         </specList>
                      </p>
-                     <p>Cutouts are treated almost similarly as <ptr target="#patches"/>. The most relevant attributes are:</p>
+                     <p></p>
+                  </div>
+                  <div xml:id="sourcesPrinted" type="div4">
+                     <head>Additional elements for Describing Printed Sources</head>
+                     <p></p>
                      <p>
                         <specList>
-                           <specDesc key="cutout" atts="removed.from"/>
-                           <specDesc key="cutout" atts="removed.by"/>
+                           <specDesc key="plateNum"/>
+                           <specDesc key="typeDesc"/>
                         </specList>
                      </p>
-                     <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (<abbr>e.g.</abbr>, <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right quarter of the page has been cut, these attributes still indicate the size of the full page (assuming that the removed section was a regular rectangle).The dimensions (<att>width</att>, <att>heigh</att>) on cutout itself are only to be used when there is a ‘gap’ in the manuscript that allows to specify the dimensions of that missing part. In this case, the bounding box dimensions are given, together with <att>x</att> and <att>y</att> to indicate the upper left point on the original page. If, however, the removed section is available by itself, then a corresponding <gi scheme="MEI">folium</gi> (or <gi scheme="MEI">bifolium</gi>) should be placed inside the cutout element, and should provide it’s own dimensions using <att>width</att> and <att>height</att> there. In this case, <att>width</att> and <att>height</att> on cutout is expendable.</p>
-                     <p>The genetic aspect of applying patches or cutting out parts of a page is described in <ptr target="#genetic"/>.</p>
+                     <p>For <gi scheme="MEI">captureMode</gi> see <ptr target="#audioAndOther"/>.</p>
                   </div>
+                  <div xml:id="addDesc" type="div4">
+                     <head>Additional Material/Information</head>
```

Comment:
"Material" and "Information" are not really "slashable"? And "Information" is very generic... can it be made more specific?

Reply:
fixed separately. Will be outdated because of revision.